### DISCUSSION TITLE
Selected Node after Deleting a Node

### DISCUSSION THREAD

**Discussion by mkraus1970** [freeplane/freeplane#1025] [Help]
I recently updated to 1.10.6 (it had been a long time since I updated on my work managed computer) and found that now when I delete a node, my "cursor" (selected node) is now the sibling to the deleted node that is centered among all the siblings.  In the version I was previously using (sorry, I'm not sure which version), when I deleted a node, my cursor was moved to the adjacent sibling node (I believe this was the next lower sibling node, unless you deleted the lowest sibling node, in which case the next higher node was selected).  I vastly prefer the old behavior.  I figured this would be customizable in preferences, but I don't see any option.  Am I missing anything?  Is there a way to change the behavior when a node is deleted?

**Comment by dpolivaev** [comment]
I have created a bug report. I am going to check it later.

**Reply by mkraus1970** [reply]
Thank you!

**Reply by dpolivaev** [reply]
@mkraus1970 I can not see the problem.
Please create an example map, attach it to the issue https://github.com/freeplane/freeplane/issues/1026 zipped and describe which node you deleted and which node was selected after.
